share/transformers/edu_ageconsearch.py

```diff
@@ -1,138 +0,0 @@
-import re
```

Comment:
Hmm, I'd kinda rather it be removed, since we'll never get new data for it, but I didn't think about the old data. I'll put it back in for now.